lsp-lens.el

```diff
@@ -375,9 +375,10 @@ CALLBACK - callback for the lenses."
   (interactive)
   (if (not lsp-lens-mode)
       (user-error "`lsp-lens-mode' not active")
```

Comment:
Those two `user-error`s can be linearized and put at the start of the function.

Reply:
Sounds good. You could the lsp-lens-mode at all. Your check will be sufficient.

Reply:
Fixed.